res/xml/he_il_1452_1.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard bottom_row="false">
+  <row>
+    <key key0="/" key2="1" key3="!" key1="esc"/>
+    <key key0="'" key2="2" key3="\@" key1=";"/>
+    <key key0="ק" key2="3" key3="\#" key1="€"/>
+    <key key0="ר" key2="4" key3="$" key1="₪"/>
+    <key key0="א" key2="5" key3="%"/>
+    <key key0="ט" key2="6" key3="^"/>
+    <key key0="ו" key2="7" key3="&amp;"/>
+    <key key0="ן" key2="8" key3="*"/>
+    <key key0="ם" key2="9" key3=")"/>
+    <key key0="פ" key2="0" key3="("/>
+    <key key0="backspace" key2="delete" key3="f11_placeholder" key4="f12_placeholder"/>
+  </row>
+  <row>
+    <key key0="ש" key1="tab" key3="`" key2="selectAll" width="1.333"/>
+    <key key0="ד" key3="´" width="1.0333"/>
+    <key key0="ג" width="1.0333"/>
+    <key key0="כ" width="1.0333"/>
+    <key key0="ע" width="1.0333"/>
+    <key key0="י" key2="-" key3="_" width="1.0333"/>
+    <key key0="ח" key2="=" key3="+" width="1.0333"/>
+    <key key0="ל" key2="]" key3="}" width="1.0333"/>
+    <key key0="ך" key2="[" key3="{" width="1.0333"/>
+    <key key0="ף" key1=":" key2="," key3="&quot;" width="1.4006"/>
+  </row>
+  <row>
+    <key key0="\\" key1="|" key2="shift" key3="loc capslock"/>
+    <key key0="ז" key1="undo" key3="redo"/>
```

Comment:
`undo`, `redo`, etc.. can be added in the settings. You don't need to specify them on the layout. Further development may make these redundant too (https://github.com/Julow/Unexpected-Keyboard/issues/233).

Reply:
k removed.